tests/scripts/expect/_common.exp

```diff
@@ -29,6 +29,7 @@
 
 proc wait_for {command expected} {
     set result 0
+    set timeout 1
```

Comment:
need to restore timeout to 10s when this proc exit?

Reply:
No. The variable will be scoped to the function.

Reply:
Thanks